src/metabase/driver.clj

```diff
@@ -348,6 +348,21 @@
 (defmethod describe-table-fks ::driver [_ _ _]
   nil)
 
+(defmulti describe-fks
+  "Returns a reducible collection of maps, each containing information about foreign keys.
+  Takes keyword arguments to narrow down the results to a set of `schema-names` or `table-names`.
+
+  Results match [[metabase.sync.interface/FastFKMetadataEntry]].
+  Results are optionally filtered by `schema-names` and `table-names` provided.
+
+  Required for drivers that support `:fast-sync-fks`."
+  {:added "0.50.0" :arglists '([driver database & {:keys [schema-names table-names]}])}
```

Comment:
Because this takes `table-names` as an argument, we can use this instead of `describe-table-fks`. That means `describe-table-fks` can be deprecated in the future.